src/apolo_app_types/protocols/stable_diffusion.py

```diff
@@ -41,7 +41,10 @@ class StableDiffusionParams(AbstractAppFieldType):
 
 
 class StableDiffusionInputs(AppInputs):
-    ingress: Ingress
+    ingress_http: IngressHttp | None = Field(
```

Comment:
and this

I mean, each app, which is mainly consumed from it's web interface (not via API), probably should always have http_ingress enabled